amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/disposition/DispositionServiceImpl.java

```diff
@@ -473,11 +473,11 @@ public NodeRef getAssociatedRecordsManagementContainer(DispositionSchedule dispo
             List<ChildAssociationRef> assocs = this.nodeService.getParentAssocs(dsNodeRef, ASSOC_DISPOSITION_SCHEDULE, RegexQNamePattern.MATCH_ALL);
             if (!assocs.isEmpty())
             {
-                if (assocs.size() != 1 && LOGGER.isWarnEnabled())
+                if (assocs.size() != 1)
                 {
                     // TODO in the future we should be able to support disposition schedule reuse, but for now just warn that
                     //      only the first disposition schedule will be considered
-                    LOGGER.warn("Retention schedule has more than one associated records management container.  " +
+                    LOGGER.atWarn().log("Retention schedule has more than one associated records management container.  " +
                             "This is not currently supported so only the first container will be considered. " +
                             "(dispositionScheduleNodeRef=" + dispositionSchedule.getNodeRef().toString() + ")");
```

Comment:
This is fine, but you can use slf4j's string construction feature:
```suggestion
                            "(dispositionScheduleNodeRef={})", dispositionSchedule.getNodeRef());
```

Reply:
done the changes as per the comment.